_posts/2022-10-28-multiple-data-source.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "Launch Highlight: Multiple Data Sources"
+authors:
+  - tyarong
+  - zengyan
+  - kroosh
+date:   2022-10-28 00:00:00 -0700
+categories:
+  - community
+---
+
+OpenSearch Dashboards' current architecture works only with a single OpenSearch cluster. To view data in different OpenSearch clusters and to navigate between Dashboards endpoints, users must have a complete OpenSearch with Dashboards stack setup. To solve this challenge, we are excited to announce for release 2.4 the multiple data sources feature. This experimental feature supports multiple data source connections in OpenSearch Dashboards.
+
+###Benefits
+The multiple data sources feature gives users benefits like:
+* **Cost Saving**: Users with hundreds of OpenSearch or Elasticsearch clusters (such as enterprise customers) don't need to create or maintain an equal number of Dashboards servers.
+* **Performance**: Dashboards runs health check against OpenSearch, and in shared-node architecture (that is, OpenSearch and Dashboards run on the same node), CPU and system memory usage can reach 10%, which can cause CPU, memory, network overhead to OpenSearch clusters. Multiple data sources can reduce the number of total running Dashboard instances and trigger health check only to its dedicated metadata storage.
```

Comment:
Change "runs health check" to "runs **a** health check."
Change "Dashboard instances" to "Dashboard**s** instances."